Assets/Scripts/PlayerMovement.cs

```diff
@@ -44,6 +44,16 @@ private void OnCollisionEnter(Collision collision)
         }
     }
 
+    private void OnTriggerEnter(Collider other)
```

Comment:
Can we put this collision event in the other one that's already defined? The one called OnCollisionEnter()? If its not possible, that's fine. Just think it might be a little cleaner.

Reply:
Unfortunately not OnCollisionEnter() is called when an object collides with another, OnTriggerEnter() is called when an object goes through a trigger volume.